src/libs/antares/study/parts/hydro/finallevelinflowsmodifyer.cpp

```diff
@@ -147,6 +148,18 @@ bool FinalLevelInflowsModifier::isActive()
             isValidLevel(initialReservoirLevel_);
 }
 
+// if the user specifies the final reservoir level, but does not specifies initial reservoir level
+// or uses wrong hydro options
+// we should inform the user that the final reservoir level wont be reached
+void FinalLevelInflowsModifier::logInfoFinLvlNotApplicable(uint year)
+{
+    if (isValidLevel(finalReservoirLevel_)
+        && (!hydro_.reservoirManagement || hydro_.useWaterValue
+            || !isValidLevel(initialReservoirLevel_)))
+        logs.info() << "Final reservoir level not applicable! Year:" << year + 1
+                    << ", Area:" << areaName_;
```

Comment:
Please tell the user why the final level is not applicable, so that they can change their input data accordingly.